provisioner/provision.js

```diff
@@ -164,7 +194,7 @@ Provisioner.prototype.runAllProvisionersOnce = function () {
   ]);
 
   p = p.then(function (res) {
-    var workerTypes = shuffle.knuthShuffle(res[0].slice(0));
+    var workerTypes = res[0];
```

Comment:
sorta... i realised it wasn't actually helping to do what I wanted to.  I might use this shuffler later on for round robining in case we decide to do random order instead of sequentially from a random order of worker names